renku_theme/login/messages/messages_en.properties

```diff
@@ -0,0 +1,508 @@
+doLogIn=Sign In
```

Comment:
It would be nice if only the strings that need to be changed were here and the rest were "inherited".